packages/cli/__tests__/zosfiles/__unit__/create/like/ds.definition.unit.test.ts

```diff
@@ -0,0 +1,36 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandDefinition } from "@zowe/imperative";
+
+describe("zos-files create  command definition", () => {
+    it("should not have changed", () => {
+        const definition: ICommandDefinition = require("../../../../../src/zosfiles/create/classicPds/ds.definition").DsDefinition;
```

Comment:
Please update the directory in path for your new handler: "../../../../../src/zosfiles/create/ds/ds.definition" (after renaming "create/like" to "create/ds" as mentioned below).

Reply:
Test here still failing because "classicPds" in the path needs to be replaced with "ds".